galley/docker/Dockerfile.galley

```diff
@@ -6,10 +6,11 @@ ARG BASE_VERSION=latest
 
 # The following section is used as base image if BASE_DISTRIBUTION=default
 FROM docker.io/istio/base:${BASE_VERSION} as default
+USER istio-proxy
```

Comment:
Upstream usage docs: ```USER <user>[:<group>] or
USER <UID>[:<GID>]```


The docs state that a `UID` may be used here instead of a specific user.  I'd recommend testing prior to line 18:

`USER 1337:1337`